packages/salesforcedx-apex-debugger/src/core/sessionService.ts

```diff
@@ -13,11 +13,11 @@ import {
 import { RequestService } from '@salesforce/salesforcedx-utils-vscode/out/src/requestService';
 
 export class SessionService {
-  private userFilter: string;
-  private requestFilter: string;
-  private entryFilter: string;
-  private project: string;
-  private sessionId: string;
+  private userFilter: string = '';
```

Comment:
Just be careful of this since we might actually check for null or undefined somewhere in the code. And with the current initializations those checks might no longer work.

Reply:
CC: @DatGuyJonathan

Reply:
I tried to look around and figure out the uses of these variables.  Some of them are handled in the builder functions like:
```
public withUserFilter(filter?: string): SessionService {
    this.userFilter = filter || '';
    return this;
}
```
but a couple of them did check for undefined elsewhere in the code, and on those I changed their type to:
```
 private _proxyUrl: string | undefined;
```

Reply:
Yup. I'm cool with the `protected mySessionService!: SessionService;`. It's just the ones where we initialize the string to `''`  that I need another pair of 👀

Reply:
The code paths for these session filter strings LGTM.